src/conversion-form/js/scripts.js

```diff
@@ -193,7 +196,7 @@ function displayValidationResult(data, textStatus, jqXHR) {
 	var interpreted = parseData(data);
 	document.getElementById("result").innerHTML = interpreted.join("<br>");
 	document.getElementById("myModal").style.display = "block";
-}
+}v
```

Comment:
Syntax error here might be leading to issues.

Reply:
Committed change, but I was actually not testing converter but rather validator when it was not working.  Still suspect issue has to do with binary files.